### PR TITLE
Preparing for 0.6 Release

### PR DESCRIPTION
**Pull request by namithj**
Preparing for 0.6 Release

# Pull Request

## What changed?

Preparing for 0.6 Release.
Change log updated.

## Why did it change?

Update version numbers to prepare for 0.6 release.

## Did you fix any specific issues?

None

## CERTIFICATION

By opening this pull request, I do agree to abide by the [Code of Conduct](https://github.com/aspirepress/.github/blob/updating-contributor-policy/CODE_OF_CONDUCT.md) and be bound by the terms of the [Contribution Guidelines](https://github.com/aspirepress/.github/blob/updating-contributor-policy/CONTRIBUTING.md) in effect on the date and time of my contribution as proven by the revision information in GitHub.

### CODE REVIEW

**Comment by namithj** [comment]
Add missing issues to changelog.

File: readme.txt [hunk 1]
```diff
@@ -1,10 +1,10 @@
 === AspireUpdate ===
 Contributors: sarah-savage, namithj, asirota
 Donate link: https://github.com/sponsors/aspirepress
-Tags: 
+Tags:
```
Comment (costdev) [inline readme.txt:4]: Should we just remove this entry? 😅

Comment (namithj) [reply readme.txt]: Removed

**Comment by costdev** [comment]
Here's a summary of changes since 0.5:

```
* Admin Settings: Notices for when settings are saved or reset
* Site Health: Add Site Health option
* Branding: Add branded notices to inform users when AspireUpdate is in operation on a screen
* Debug: Add Clear Logs and View Logs functionality
* I18N: Add Spanish translation
* I18N: Add Catalan translation
* I18N: Add Dutch translation
* I18N: Add Swedish translation
* I18N: Update German translation
* I18N: Update Dutch translation
* I18N: Update French translation
* Testing: Add Git Updater integration
* Testing: Make Playground default to the `main` branch
* Testing: Support both main and playground-ready links in the README
* Testing: Remove Hello Dolly from the Playground blueprint
* Security: Fix Plugin Check security warnings
```

**Comment by asirota** [comment]
Why don't we move the tag to a 1.0b1 instead and then mark as 1.0 when we ship?

File: readme.txt [hunk 2]
```diff
@@ -34,7 +33,25 @@ Answer to foo bar dilemma.
 
 == Changelog ==
 
+= 0.6 =
+* Admin Settings: Added notices for when settings are saved or reset
+* Site Health: Added Site Health option
```
Comment (costdev) [inline readme.txt:38]: Oops, this one is actually part of the issue template changes and can be removed from the changelog. My mistake.

**Comment by namithj** [comment]
> Why don't we move the tag to a 1.0b1 instead and then mark as 1.0 when we ship?

Not sure we can call it until we actually test with AC.

File: readme.txt [hunk 3]
```diff
@@ -34,7 +33,24 @@ Answer to foo bar dilemma.
 
 == Changelog ==
 
+= 0.6 =
+* Admin Settings: Added notices for when settings are saved or reset
+* Branding: Added branded notices to inform users when AspireUpdate is in operation on a screen
+* Debug: Added Clear Logs and View Logs functionality
```
Comment (costdev) [inline readme.txt:39]:
```suggestion
* Multisite: Added multisite support
* Debug: Added Clear Logs and View Logs functionality
```

File: readme.txt [hunk 4]
```diff
@@ -34,7 +33,25 @@ Answer to foo bar dilemma.
 
 == Changelog ==
 
+= 0.6 =
+* Admin Settings: Added notices for when settings are saved or reset
+* Branding: Added branded notices to inform users when AspireUpdate is in operation on a screen
+* Multisite: Added multisite support
```
Comment (asirota) [inline readme.txt:39]: cool it's there..

**Pull request merged**